### DISCUSSION TITLE
v2.0.0-alpha.64

### DISCUSSION THREAD

**Discussion by leaanthony** [wailsapp/wails#673] [Releases]
* ANSI encoding support for Tray/Menu Items
* Added Mac OS detection
* Initial support for desktop notifications
* Support pass-thru tags
* Improved templates
* Font smoothing for vanilla app

<hr /><em>This discussion was created from the release <a href='https://github.com/wailsapp/wails/releases/tag/v2.0.0-alpha.64'>v2.0.0-alpha.64</a>.</em>

**Comment by Purple-CSGO** [comment]
Hello @leaanthony,
  I've kept watching this repo for quite some time. You've been releasing very frequently~
  I'm really looking forward to wails v2. Could you estimate when v2.0 finally releases?
  And,,, will it support vue3 or actually does?

**Reply by leaanthony** [reply]
Hi. No estimate yet, apart from "this year - hopefully". Due 3 will be supported, yes. It's supported in v1, just not on Windows. But that's a due 3 limitation. Also, the more people who get involved, the quicker it'll be 👍
